roles/ksql/tasks/main.yml

```diff
@@ -147,7 +147,7 @@
     kerberos_keytab_path: "{{ksql_kerberos_keytab_path}}"
     kerberos_keytab_destination_path: "{{ksql_keytab_path}}"
     kerberos_handler: "restart ksql"
-  when: (kafka_broker_listeners[ksql_kafka_listener_name]['sasl_protocol'] | default(sasl_protocol) | confluent.platform.normalize_sasl_protocol == 'GSSAPI') or (kafka_broker_listeners[ksql_processing_log_kafka_listener_name]['sasl_protocol'] | default(sasl_protocol) | confluent.platform.normalize_sasl_protocol == 'GSSAPI' and ksql_log_streaming_enabled | bool)
+  when: "'GSSAPI' in (kafka_broker_listeners[ksql_kafka_listener_name]['sasl_protocol'] | default(sasl_protocol) | confluent.platform.normalize_sasl_protocol) or ('GSSAPI' in (kafka_broker_listeners[ksql_processing_log_kafka_listener_name]['sasl_protocol'] | default(sasl_protocol) | confluent.platform.normalize_sasl_protocol) and ksql_log_streaming_enabled | bool)"
```

Comment:
I think `ksql_log_streaming_enabled` was inside the second or statement.

Reply:
It is still inside second or statement.